src/content/news/2025-04-04-maplibre-newsletter-march-2025/index.mdx

```diff
@@ -0,0 +1,69 @@
+
+With that note, let’s dive into this month’s highlights.
+
+## MapLibre Native
+
+- [Grab](https://en.wikipedia.org/wiki/Grab_Holdings) is [contributing](https://github.com/maplibre/maplibre-native/pulls?q=author%3Agrab-liyanjin) to MapLibre Native. While they currently use an internal fork, they plan to adopt MapLibre Native later this year. We’re looking forward to collaborating with them and supporting their onboarding process.
+
+- MapLibre iOS [6.12.1](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.1) and MapLibre Android [11.8.3](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.3), [11.8.4](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.4) and [11.8.5](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.5) were released. We’ve also started publishing the core as a [static library](https://github.com/maplibre/maplibre-native/releases?q=core-&expanded=true) to support the [Rust wrapper](https://crates.io/crates/maplibre_native) currently in development—note that there are no API stability guarantees at this stage. If you have a use case for this, [let us know](https://github.com/maplibre/maplibre-native/discussions/categories/show-and-tell) about it.
+
+- Some neat improvements are in the pipeline for MapLibre Native, including:
+  - [Level-of-detail controls](https://github.com/maplibre/maplibre-native/pull/2958), to limit tile loading in scenarios like navigation.
+  - A toggleable [rendering HUD](https://github.com/maplibre/maplibre-native/pull/3322) for quick access to performance stats, especially useful on embedded hardware where getting these numbers through other means may be difficult.
+  - Support for [MapLibre Tiles](https://github.com/maplibre/maplibre-native/pull/3246), offering better performance, improved battery life, and reduced bandwidth usage.
+
+<div style={{ textAlign: "center" }}>
+  <Image
+    src={hudScreenshot}
+    style={{
+      maxWidth: "300px",
+      maxHeight: "600px",
+      width: "100%",
+      height: "auto",
+      display: "inline-block",
+    }}
+    alt="Rendering HUD screenshot"
+  />
+  <p
+    style={{
+      fontSize: "0.9rem",
+      color: "#555",
+      marginTop: "8px",
+      fontStyle: "italic",
+    }}
+  >
+    <em>Rendering HUD screenshot</em>
+  </p>
+</div>
+
+## Web
+
+We released two minor versions of MapLibre GL JS this month: [v5.2.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.2.0) and [v5.3.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.3.0), both focused on bug fixes and small enhancements. Work is underway on two exciting features: supporting fonts via system fonts, as an alternative to the glyphs field and introducing a global map state. These additions will make it easier for style authors to design and theme maps more intuitively. Big thanks to all the contributors for helping push these improvements forward - stay tuned for what’s next!
```

Comment:
> introducing a global map state

We should link to https://github.com/maplibre/maplibre-style-spec/pull/1044 and mention the discussion how/if to implement this the discussion is still ongoing.

Reply:
Yeah, I agree, map state discussion is a bit different from when I wrote this...